Alicia/feat/cli

abrooks11 opened the pull request:
**What is the problem you were trying to solve?**
Wanted to reduce ramp up time between installing the dependency and caching responses 


**What is your solution and why did you solve this problem the way you did?**
Added a cli command, "npx quell init",  that installs necessary dependencies and added a quell.config file


**Provide any additional notes on testing this functionality:**



**Screenshots / gifs of working solution:**

abrooks11 committed:
added boiler plate files for cli command

abrooks11 committed:
feat(CLI) added running CLI command for quell set-up

abrooks11 committed:
Merge branch 'dev' into alicia/feat/cli

abrooks11 committed:
removed dependency

abrooks11 committed:
Merge branch 'dev' into alicia/feat/cli

abrooks11 committed:
fix: added declaration support to package and ts.config

jeremycoledalton reviewed (changes requested):
Make sure the import statements are connecting to the correct locations

figure out the import for graphqlHTTP

MAYBE adjust type errors.

abrooks11 committed:
fix:
- update file path for import statements
- replaced local host with address
- create parent folders for example server and and schema files
- update templates for quell-config and env
- add types to example schema
- refactor example server template

abrooks11 commented:
Fix should resolve broken import statements and type errors. CLI --example option creates example server and schema files. 
- update templates for quell-config and env
- replaced local host with address
- refactor example server template

abrooks11 committed:
feat: add cacheHit: boolean to quellRouter responses

abrooks11 committed:
feat: add cacheHit: boolean to QuellCache
